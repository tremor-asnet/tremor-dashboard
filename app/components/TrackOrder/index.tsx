import { Bold, Card } from "@tremor/react";

// Icon React
import { IoMdCheckmark } from "react-icons/io";

// Components
import { StepOrder } from "@/components";

// Types
import { StepOrderProps } from "@/types";

interface TrackOrderProps {
  data: StepOrderProps[];
  descriptionInfo: string;
  deliveredAt: string;
}

const TrackOrder = ({
  data,
  descriptionInfo,
  deliveredAt,
}: TrackOrderProps) => {
  return (
    <Card className="h-full bg-tremor-primary dark:bg-dark-tremor-primary p-0 border-none ring-0 pt-6">
      <Bold className="dark:text-white mb-4 pl-6">Track order</Bold>
      <ul className="relative m-0 w-full p-0 pb-6">
        {data.map(item => {
          const { iconInfo, titleInfo } = item;
          return (
            <>
              <StepOrder
                className="after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[1px] after:h-[calc(100%-2.25rem)] after:w-[0.125rem] after:bg-border after:content-['']"
                backgroundColor="bg-light"
                iconInfo={iconInfo}
                titleInfo={titleInfo}
                descriptionInfo={descriptionInfo}
              />
            </>
          );
        })}
        <StepOrder
          backgroundColor="bg-few"
          iconInfo={<IoMdCheckmark />}
          titleInfo="Order delivered"
          descriptionInfo={deliveredAt}
        />
      </ul>
    </Card>
  );
};

export default TrackOrder;
