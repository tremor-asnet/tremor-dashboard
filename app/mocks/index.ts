export * from "./card";
export * from "./profile";
export * from "./project";
export * from "./charts";
export * from "./analytics";
export * from "./sales";
export * from "./table-list";
export * from "./orders";
export * from "./product";
