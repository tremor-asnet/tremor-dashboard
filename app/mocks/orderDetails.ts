import { TrackOrderProps } from "@/types";

export const mockInvoiceHeader = {
  id: 241342,
  createdAt: "2024-01-11T01:48:52Z",
  orderCode: "KF332",
};

export const mockOrderSummary = {
  productPrice: 90,
  delivery: 14,
  taxes: 1.95,
};

export const mockTrackOrder: TrackOrderProps = {
  id: "#1832412",
  generateOrderAt: "2024-01-11T02:48:52Z",
  deliveredAt: "2024-01-11T02:48:52Z",
  transmitedToCourierAt: "2024-01-11T02:48:52Z",
  generateOrderId: "2024-01-11T02:48:52Z",
};
