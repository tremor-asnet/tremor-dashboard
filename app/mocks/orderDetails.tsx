import { FaBell } from "react-icons/fa";
import { MdInventory } from "react-icons/md";
import { IoCart } from "react-icons/io5";
import { StepOrderProps } from "@/types/orderDetail";

export const mockInvoiceHeader = {
  id: 241342,
  createdAt: "2024-01-11T01:48:52Z",
  orderCode: "KF332",
};

export const mockTrackOrder: StepOrderProps[] = [
  {
    iconInfo: <FaBell />,
    titleInfo: "Order received",
  },
  {
    iconInfo: <MdInventory />,
    titleInfo: "Generate order id",
  },
  {
    iconInfo: <IoCart />,
    titleInfo: "Order transmited to courier",
  },
];
