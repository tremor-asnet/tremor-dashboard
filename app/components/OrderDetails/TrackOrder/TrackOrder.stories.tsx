import type { Meta, StoryObj } from "@storybook/react";

// Components
import TrackOrder from "./TrackOrder";

// Mocks
import { mockTrackOrder } from "@/mocks/orderDetails";

// Mock data

const meta = {
  title: "Components/TrackOrder",
  component: TrackOrder,
  tags: ["autodocs"],
} as Meta<typeof TrackOrder>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  render: () => (
    <TrackOrder
      id={mockTrackOrder.id}
      generateOrderAt={mockTrackOrder.generateOrderId}
      deliveredAt={mockTrackOrder.deliveredAt}
      transmitedToCourierAt={mockTrackOrder.transmitedToCourierAt}
      generateOrderId={mockTrackOrder.generateOrderId}
    />
  ),
};
