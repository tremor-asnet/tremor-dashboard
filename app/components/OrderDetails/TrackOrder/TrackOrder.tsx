// Components
import { Bold, Flex } from "@tremor/react";

// Icon React
import { FaBell } from "react-icons/fa";
import { IoMdCheckmark } from "react-icons/io";
import { IoCart } from "react-icons/io5";
import { MdInventory } from "react-icons/md";
import { RxCross2 } from "react-icons/rx";
import { RiRefund2Line } from "react-icons/ri";

// Types
import { TrackOrderProps } from "@/types/orderDetails";

// Helpers
import { formatDateTime } from "@/helpers";

const TrackOrder = ({
  id,
  transmitedToCourierAt,
  generateOrderId,
  deliveredAt,
  generateOrderAt,
  status = 0,
}: TrackOrderProps) => {
  const trackingOrder = (status: number, date: string) => {
    switch (status) {
      case 0:
        return (
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-few text-sm font-medium text-dark-primary">
              <IoMdCheckmark />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order delivered
              </span>
              <span className="text-tremor-label text-secondary uppercase font-light dark:text-dark-romance">
                {formatDateTime(date)}
              </span>
            </div>
          </div>
        );
      case 1:
        return (
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-red-500 text-sm font-medium text-dark-primary">
              <RxCross2 />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order canceled
              </span>
              <span className="text-tremor-label text-secondary uppercase font-light dark:text-dark-romance">
                {formatDateTime(date)}
              </span>
            </div>
          </div>
        );

      case 2:
        return (
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-light text-lg font-medium text-dark-primary">
              <RiRefund2Line />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order refunded
              </span>
              <span className="text-tremor-label text-secondary uppercase font-light dark:text-dark-romance">
                {formatDateTime(date)}
              </span>
            </div>
          </div>
        );

      default:
        return (
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-few text-sm font-medium text-dark-primary">
              <IoMdCheckmark />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order delivered
              </span>
              <span className="text-tremor-label text-secondary uppercase font-light dark:text-dark-romance">
                {formatDateTime(deliveredAt)}
              </span>
            </div>
          </div>
        );
    }
  };

  return (
    <>
      <Bold className="text-primary font-semibold capitalize dark:text-white tracking-[0.12px]">
        Track order
      </Bold>
      <ul className="relative m-0 w-full p-0 pb-6 tracking-[0.4px]">
        <li className="relative h-fit after:absolute after:left-[1.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:line-content">
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <FaBell />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order received
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {formatDateTime(generateOrderAt)}
              </span>
            </div>
          </div>
        </li>

        <li className="relative h-fit after:absolute after:left-[1.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:line-content">
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <MdInventory />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Generate order id {id}
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {formatDateTime(generateOrderId)}
              </span>
            </div>
          </div>
        </li>

        <li className="1234 relative h-fit after:absolute after:left-[1.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:content-empty">
          <div className="flex cursor-pointer items-start pl-2 pr-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 min-w-[32px] mt-1 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <IoCart />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order transmited to courier
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {formatDateTime(transmitedToCourierAt)}
              </span>
            </div>
          </div>
        </li>

        <li className="relative h-fit">{trackingOrder(status, deliveredAt)}</li>
      </ul>
    </>
  );
};

export default TrackOrder;
