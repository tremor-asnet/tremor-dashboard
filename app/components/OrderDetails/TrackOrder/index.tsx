import { Card, Flex } from "@tremor/react";
import dayjs from "dayjs";

// Icon React
import { FaBell } from "react-icons/fa";
import { IoMdCheckmark } from "react-icons/io";
import { IoCart } from "react-icons/io5";
import { MdInventory } from "react-icons/md";

// Types
import { TrackOrderProps } from "@/types/orderDetails";

const TrackOrder = ({
  id,
  transmitedToCourierAt,
  generateOrderId,
  deliveredAt,
  generateOrderAt,
}: TrackOrderProps) => {
  return (
    <Card className="h-full bg-tremor-primary dark:bg-dark-tremor-primary p-0 border-none ring-0">
      <ul className="relative m-0 w-full p-0 pb-6">
        <li className="relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:line-content">
          <Flex className="justify-start px-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <FaBell />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order received
              </span>
              <span className="text-tremor-label text-primary font-light dark:text-dark-romance">
                {dayjs(generateOrderAt).format("DD MMM h:m A")}
              </span>
            </div>
          </Flex>
        </li>

        <li className="relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:line-content">
          <div className="flex cursor-pointer items-center px-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <MdInventory />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Generate order id {id}
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {dayjs(generateOrderId).format("DD MMM h:m A")}
              </span>
            </div>
          </div>
        </li>

        <li className="1234 relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[5px] after:line-track after:w-[0.125rem] after:bg-border after:content-empty">
          <div className="flex cursor-pointer items-center px-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary">
              <IoCart />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order transmited to courier
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {dayjs(transmitedToCourierAt).format("DD MMM h:m A")}
              </span>
            </div>
          </div>
        </li>

        <li className="relative h-fit">
          <div className="flex cursor-pointer items-center px-6 pt-6 leading-[1.3rem] after:line-content">
            <span className="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-few text-sm font-medium text-dark-primary">
              <IoMdCheckmark />
            </span>
            <div className="flex flex-col">
              <span className="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] dark:text-dark-primary">
                Order delivered
              </span>
              <span className="text-tremor-label text-primary uppercase font-light dark:text-dark-romance">
                {dayjs(deliveredAt).format("DD MMM h:m A")}
              </span>
            </div>
          </div>
        </li>
      </ul>
    </Card>
  );
};

export default TrackOrder;
