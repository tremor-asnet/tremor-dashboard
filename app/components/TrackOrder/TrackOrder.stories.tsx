import type { Meta, StoryObj } from "@storybook/react";

// Components
import TrackOrder from ".";

// Mocks
import { mockTrackOrder } from "@/mocks/orderDetails";

// Mock data

const meta = {
  title: "Components/TrackOrder",
  component: TrackOrder,
  tags: ["autodocs"],
} as Meta<typeof TrackOrder>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  render: () => (
    <TrackOrder
      data={mockTrackOrder}
      deliveredAt={"27 , JAN"}
      descriptionInfo=""
    />
  ),
};
