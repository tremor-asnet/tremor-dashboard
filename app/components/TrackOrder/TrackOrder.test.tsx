import { render } from "@testing-library/react";

// Components
import TrackOrder from ".";

// Mocks
import { mockTrackOrder } from "@/mocks/orderDetails";

const TrackOrderProps = {
  data: mockTrackOrder,
  deliveredAt: "27 , JAN",
  descriptionInfo: "Order Deliver",
};

const TrackOrderComponent = () => render(<TrackOrder {...TrackOrderProps} />);

describe("TrackOrder Component", () => {
  test("render TrackOrder component with snapshot correctly", () => {
    const { container } = TrackOrderComponent();

    expect(container).toMatchSnapshot();
  });
});
