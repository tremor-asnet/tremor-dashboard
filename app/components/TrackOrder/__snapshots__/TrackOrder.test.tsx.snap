// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackOrder Component render TrackOrder component with snapshot correctly 1`] = `
<div>
  <div
    class="tremor-Card-root relative w-full text-left rounded-tremor-default ring-tremor-ring shadow-tremor-card dark:ring-dark-tremor-ring dark:shadow-dark-tremor-card border-tremor-brand dark:border-dark-tremor-brand h-full bg-tremor-primary dark:bg-dark-tremor-primary p-0 border-none ring-0 pt-6"
  >
    <b
      class="text-inherit font-bold dark:text-white mb-4 pl-6"
    >
      Track order
    </b>
    <ul
      class="relative m-0 w-full p-0 pb-6"
    >
      <li
        class="relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[1px] after:h-[calc(100%-2.25rem)] after:w-[0.125rem] after:bg-border after:content-['']"
      >
        <div
          class="flex justify-start px-6 pt-6 leading-[1.3rem] after:content-['']"
        >
          <span
            class="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224 512c35.32 0 63.97-28.65 63.97-64H160.03c0 35.35 28.65 64 63.97 64zm215.39-149.71c-19.32-20.76-55.47-51.99-55.47-154.29 0-77.7-54.48-139.9-127.94-155.16V32c0-17.67-14.32-32-31.98-32s-31.98 14.33-31.98 32v20.84C118.56 68.1 64.08 130.3 64.08 208c0 102.3-36.15 133.53-55.47 154.29-6 6.45-8.66 14.16-8.61 21.71.11 16.4 12.98 32 32.1 32h383.8c19.12 0 32-15.6 32.1-32 .05-7.55-2.61-15.27-8.61-21.71z"
              />
            </svg>
          </span>
          <div
            class="flex flex-col"
          >
            <span
              class="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] after:content-[data-content] dark:text-dark-primary"
            >
              Order received
            </span>
            <span
              class="text-tremor-label text-primary font-light dark:text-dark-romance"
            >
              Order Deliver
            </span>
          </div>
        </div>
      </li>
      <li
        class="relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[1px] after:h-[calc(100%-2.25rem)] after:w-[0.125rem] after:bg-border after:content-['']"
      >
        <div
          class="flex justify-start px-6 pt-6 leading-[1.3rem] after:content-['']"
        >
          <span
            class="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0V0z"
                fill="none"
              />
              <path
                d="M20 2H4c-1 0-2 .9-2 2v3.01c0 .72.43 1.34 1 1.69V20c0 1.1 1.1 2 2 2h14c.9 0 2-.9 2-2V8.7c.57-.35 1-.97 1-1.69V4c0-1.1-1-2-2-2zm-5 12H9v-2h6v2zm5-7H4V4l16-.02V7z"
              />
            </svg>
          </span>
          <div
            class="flex flex-col"
          >
            <span
              class="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] after:content-[data-content] dark:text-dark-primary"
            >
              Generate order id
            </span>
            <span
              class="text-tremor-label text-primary font-light dark:text-dark-romance"
            >
              Order Deliver
            </span>
          </div>
        </div>
      </li>
      <li
        class="relative h-fit after:absolute after:left-[2.45rem] after:top-[3.6rem] after:mt-[1px] after:h-[calc(100%-2.25rem)] after:w-[0.125rem] after:bg-border after:content-['']"
      >
        <div
          class="flex justify-start px-6 pt-6 leading-[1.3rem] after:content-['']"
        >
          <span
            class="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-light text-sm font-medium text-dark-primary"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="176"
                cy="416"
                r="32"
              />
              <circle
                cx="400"
                cy="416"
                r="32"
              />
              <path
                d="M456.8 120.78a23.92 23.92 0 00-18.56-8.78H133.89l-6.13-34.78A16 16 0 00112 64H48a16 16 0 000 32h50.58l45.66 258.78A16 16 0 00160 368h256a16 16 0 000-32H173.42l-5.64-32h241.66A24.07 24.07 0 00433 284.71l28.8-144a24 24 0 00-5-19.93z"
              />
            </svg>
          </span>
          <div
            class="flex flex-col"
          >
            <span
              class="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] after:content-[data-content] dark:text-dark-primary"
            >
              Order transmited to courier
            </span>
            <span
              class="text-tremor-label text-primary font-light dark:text-dark-romance"
            >
              Order Deliver
            </span>
          </div>
        </div>
      </li>
      <li
        class="relative h-fit undefined"
      >
        <div
          class="flex justify-start px-6 pt-6 leading-[1.3rem] after:content-['']"
        >
          <span
            class="mr-3 flex h-8 w-8 items-center justify-center rounded-full bg-few text-sm font-medium text-dark-primary"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M186.301 339.893L96 249.461l-32 30.507L186.301 402 448 140.506 416 110z"
              />
            </svg>
          </span>
          <div
            class="flex flex-col"
          >
            <span
              class="text-tremor-default text-primary font-semibold after:absolute after:flex after:text-[0.8rem] after:content-[data-content] dark:text-dark-primary"
            >
              Order delivered
            </span>
            <span
              class="text-tremor-label text-primary font-light dark:text-dark-romance"
            >
              27 , JAN
            </span>
          </div>
        </div>
      </li>
    </ul>
  </div>
</div>
`;
